google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -270,6 +282,10 @@ public void setAdminHost(String adminHost) {
 
   @Override
   public Integer getAdminPort() {
+    Integer adminPort = settings.getAdminPort();
+    if (adminPort == null) {
```

Comment:
Maybe set these defaults in AppEngineModelSettings? Like how `port` is currently being set there to 8080.